static/src/javascripts/projects/common/modules/commercial/dfp.js

```diff
@@ -236,6 +250,19 @@ define([
         postDisplay = function () {
             mediator.on('window:resize', windowResize);
         },
+        showAdsFreeSurvey = function (size, $adSlot) {
+            fastdom.write(function () {
+                var crossIcon = svgs('crossIcon'),
+                    dataAttr = isAdfreeSurvey('variant') ? 'hide ads' : 'hide ads simple',
+                    $adSlotRemove = $(document.createElement('div')).addClass('ad-slot--remove').attr('data-link-name', dataAttr)
```

Comment:
We hang everything off of a class named .ad-slot--remove, but looking at the CSS, I'm not sure how much of what that class relates purely to removing an ad, and how much of it relates specifically to our ad free survey. If it's more the latter than the former, it might be wise if the class name reflects that.